envoy/buffer/buffer.h

```diff
@@ -480,7 +480,6 @@ class Instance {
 using InstancePtr = std::unique_ptr<Instance>;
 
 /**
- * A factory for creating buffers which call callbacks when reaching high and low watermarks.
```

Comment:
I think you should keep this class comment

Reply:
Whoops, done!